src/ActiveTrail/Campaign/EmailCampaign.php

```diff
@@ -28,22 +28,35 @@ class EmailCampaign extends CampaignBase implements EmailCampaignInterface {
 
   /**
    * EmailCampaign constructor.
-   * @param $apiToken
+   *
+   * @param $api_token
    */
-  public function __construct($apiToken) {
-    parent::__construct($apiToken, EndPoints::$CAMPAIGNS_CONTACTS);
+  public function __construct($api_token) {
+    parent::__construct($api_token, EndPoints::$CAMPAIGNS_CONTACTS);
     // Initialize payload structure
     $updateContainer = new ApiCampaignUpdateContainer('', new ApiCampaignDetails(), new ApiCampaignDesign(),
       new ApiCampaignTemplate(), new ApiABSettings(), new ApiEcommerceDataList(), []);
     $this->campaignPayload = new ApiCampaignContactPost($updateContainer, new ApiCampaignContact());
   }
 
-  public function setSubject ($subject) {
+  /**
+   * Sets the email's subject.
+   * @param $subject
+   *
+   * @return $this
```

Comment:
`$this` is not a type and as such, can not be the type that is going to be returned.

use `@return static` instead